### ISSUE TITLE
Wiiflow Lite breaks on newer devkitpro revisions >r30 

### ISSUE THREAD

**Issue by Ntemis** [dborth/dosbox-wii#51]
wiiflow lite is compiled with ppc r30. if compiled with r33 then the plugin for dosboxwii doesn't work correctly. not sure why, but it does work correctly if wiiflow lite is compiled with r30. so author is forced to freeze at using r30. Since you are the author can you please have a look if there is anything to be done?
For reference to the issue here is a link:
https://github.com/Fledge68/WiiFlow_Lite/issues/52

**Comment by aaaaaaaaaaaaas** [comment]
yeah that isnt going to happen unless you do it yourself, the last update was in June 2012. this project is LONG dead
